homeassistant/components/rest/entity.py

```diff
@@ -80,3 +72,24 @@ async def async_update(self):
     @abstractmethod
     def _update_from_rest_data(self):
         """Update state from the rest data."""
+
+
+class RestEntity(BaseRestEntity):
```

Comment:
Is this temporary until we have migrated the binary sensor platform, or do we keep it for custom integrations that may rely on it?

Reply:
It's for the `rest` integration's `binary_sensor` platform.